data/faq/application_template.md

```diff
@@ -0,0 +1,122 @@
+------------
+
+- Which project from http://projects.coala.io are you applying for?
+  If you have your own idea, please add it to our projects list:
+  https://github.com/coala/projects/blob/master/data/projects.js
+
+- How is your project helping coala and its community?
+
+- What is the final goal for this project? What would make it a total
+  and perfect success?
+
+- Are you already engaged with the project's possible mentors and do
+  you have any preference for a particular mentor?
+
+- What parts of coala do you have to work with in order to complete
+  this project? What else are you planning on using?
+
+- Why are you the right person to work on this project?
+
+- How do you plan to achieve completion of your project?
+    - Please provide a schedule with dates and important
+      milestones/deliverables (preferably in two week increments).
+      This is a great help in evaluating your own progress and helps you
+      achieving your goal as it forces you to plan your work preemptively.
+
+Add all ideas you have to solve the project here. This could include
+research on linters, libraries and similar problems as well as mockups
+of code, api and ui changes.
+
+
+Other Commitments
+-----------------
+
+- Do you have any other commitments during the GSoC period,
+  May 8th to August 29th?
+
+    - We don't penalize students for needing adjustments to schedule if
+      they're up-front about them and have a plan to mitigate any issues.
+      However, we *will* fail students for lying about their availability
+      and subsequently falling behind in their work. Be honest!
+
+    - Do you have exams or classes that overlap with this period?
+
+    - Do you have plans to have any other job or internship during this
+      period?
+        - This is __NOT RECOMMENDED AT ALL__ as GSoC is intended to be a
+        full time job. But can be worked around if it only a small overlap.
+
+    - Do you have any other short term commitments during this period?
+        - e.g. family wedding, conferences, volunteer projects, planned
+          vacation days
+
+- Have you applied with any other organizations? If so, do you have a
+  preferred project/org? (This will help us in the event that more than one
+  org decides they wish to accept your proposal.)
+
+
+Extra Information
+-----------------
+
+This additional information isn't needed by coala but can help us to learn
+more about you. All fields in this section are optional.
+
+- Link to openhub profile:
+  (https://openhub.net is a nice way to showcase all your oss contributions)
+- University info
+    - University name:
+    - Major:
+    - Current year and expected graduation date:
+    - Degree:
+- Other Contact info:
+    - Alternate contact info in case primary mail above stops working:
+    - Homepage:
+    -Instant messaging:
+
+- Will you be able to join us for a conference such as EuroPython or GUADEC
+  and present your work if at least partial sponsorship will be provided?
```

Comment:
I think we shouldnt support conference attendence during their project; it should only occur afterwards.
(I've been burnt by midterm conferences.) Better to be clear.